recipe/meta.yaml

```diff
@@ -7,22 +7,31 @@ package:
 source:
   fn: v{{ version }}.tar.gz
   url: https://github.com/gflags/gflags/archive/v{{ version }}.tar.gz
-  sha1: 8bdbade9d041339dc14b4ab426e2354a5af38478
+  sha1: bf07a7926cb2ce4dd4048bdc081f9e00e91cefd6
 
 build:
   number: 0
-  skip: true  # [win]
```

Comment:
Please keep, but change like so.

```yaml
  skip: true  # [win and py==36]
```

Note that Python 3.5 and 3.6 use the same VC version. So there is no need to build both. 😄